ow_plexil/src/plexil-adapter/PlexilPlanner.h

```diff
@@ -0,0 +1,32 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+
+#ifndef Plexil_Planner_H
+#define Plexil_Planner_H
+
+#include <ros/ros.h>
+#include <ow_plexil/PlannerCommand.h>
+#include "OwExecutive.h"
+#include "OwInterface.h"
+
+class PlexilPlanner{
+  public:
+    PlexilPlanner();
+    void plannerCommandsCallback(const ow_plexil::PlannerCommand::ConstPtr&);
+    void start();
+    void initialize();
+
+  
+  private:
+    std::unique_ptr<ros::NodeHandle> m_genericNodeHandle;
+    std::unique_ptr<OwExecutive> m_executive;
+    std::unique_ptr<ros::Subscriber> m_plannerCommandSubscriber;
+    std::unique_ptr<ros::Publisher> m_planStatusPublisher;
+    std::vector<std::string> plan_array;
+    bool first_plan;
```

Comment:
Per our coding guidelines member variables should be prefixed "m_", so m_first _plan.